libnavui-maps/src/main/res/drawable/mapbox_bg_button.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    <solid android:color="#ffffff" />
+    <stroke android:color="#CDD0D0" android:width="2dp" />
```

Comment:
Border color and width should also be a customizable attribute.

Reply:
Developers can customize any background attributes by providing their own background drawable.